k9mail/src/main/AndroidManifest.xml

```diff
@@ -65,6 +65,8 @@
         android:icon="@drawable/icon"
         android:label="@string/app_name"
         android:theme="@style/Theme.K9.Startup"
+        android:resizeableActivity="true"
+        android:supportsPictureInPicture="true"
```

Comment:
According to the documentation `supportsPictureInPicture` needs to be specified on an Activity level. However, K-9 Mail neither supports picture in picture mode nor is it a video player app. So we don't want to set this attribute in our manifest.